transform/mattermost-analytics/models/marts/nps/_nps__models.yml

```diff
@@ -0,0 +1,33 @@
+version: 2
+
+models:
+  - name: dim_nps_feedback
+    description: |
+      List of known customers with active on prem licenses.
+
+    columns:
+      - name: server_id
+        description: The ID of the server the event originated from..
+        tests:
+          - not_null
+      - name: user_id
+        description: The ID of the user that sent the event.
+      - name: license_id
+        description: The license id.
+      - name: feedback
+        description: NPS feedback submitted.
+      - name: score
+        description: NPS score submitted.
+      - name: server_version
+        description: The Server's version.
+      - name: installation_type
+        description: The installation type of the server.        
```

Comment:
might alos help document possible values for the type.

Reply:
I updated the logic here to utilize the license_id directly from NPS instead of joining with other models.

Reply:
I wrote a `CASE WHEN` statement, IMO this would work, but please let me know what you think

Reply:
Removed this, as we'll be doing the join outside in Looker and the python code.